templates/default/vlan/vlanlist.html

```diff
@@ -33,18 +37,21 @@ <h1>{$pagetitle}</h1>
     </thead>
     <tbody>
         {foreach $vlanlist as $v}
-        <tr>
+        <tr class="highlight">
             <td>
                 {$v.id}
             </td>
             <td>
                 {$v.vlanid}
             </td>
-            <td>
-                {$v.description}
+            <td {tip text="{$v.description}"}>
```

Comment:
Tu wystarczy:
```smarty
<td {tip text=$v.description}>
```
Przy okazji? Sprawdzałeś, czy w pole description vlanu można wpisać np.:
```js
<script>alert('debian jest do kitu');</script>
```
i czy to nie generuje XSS? ;-)

Reply:
Czy na pewno będzie działać bez "ciapków" ?
Tak jest podatne - ja nadal liczę że wprowadzisz jakąś zmianę ale systemową.

Reply:
Poprawione: https://github.com/lmsgit/lms/pull/1890/commits/688746380c0af45806233e469b2394d897cd2111

XSS tu: https://github.com/lmsgit/lms/pull/1890/commits/0ebd0f092182570f6c14ac66110211900e7becb0